src/drivers/camera_trigger/camera_trigger.cpp

```diff
@@ -765,7 +789,7 @@ CameraTrigger::engage(void *arg)
 	struct camera_trigger_s	trigger = {};
 
 	// Set timestamp the instant after the trigger goes off
-	trigger.timestamp = hrt_absolute_time();
+	trigger.timestamp = trig->_trigger_timestamp;
```

Comment:
And then I'd use `now` here. I know it's the same but I think it would be more understandable.